_data/simple-icons.json

```
@@ -8198,6 +8198,11 @@
             "hex": "CD1414",
             "source": "https://www.penny.de/"
         },
+        {
+            "title": "penpot",
+            "hex": "000000",
+            "source": "https://penpot.app/"
```

According to the discussion in #6664, I believe you've used the SVG file on GitHub as the source for converting the icon. If that's the case, I think that URL should be used instead.

```suggestion
            "source": "https://github.com/penpot/penpot/blob/cbe51fcabd8146391df007254b2a854bae857fe6/frontend/resources/images/icons/penpot-logo-icon.svg"
```

---

I extracted the logo from the official website instead of github by using svg grubber as mentioned in PR.

---

Ah, sorry for my oversight. You're right, it's mentioned in the PR. :+1:

---

Please have a look at https://github.com/simple-icons/simple-icons/pull/7911#issuecomment-1268152756